thewalrus/quantum.py

```diff
@@ -713,3 +716,68 @@ def is_classical_cov(cov, hbar=2, atol=1e-08):
         if np.all(vals >= 0):
             return True
     return False
+
+
+def gen_single_mode_dist(s, nmax=50, N=1):
+    """Generate the photon number distribution of N identical single mode squeezed states with squeezing parameter s up to cutoff nmax

```

Comment:
Can keep the first sentence of the docstring short and concise, since the parameters are described below.

```suggestion
    """Generate the photon number distribution of :math:`N` identical single mode squeezed states.
    
```